examples/html-styled/Main.elm

```diff
@@ -82,9 +82,9 @@ legacyBorderRadius amount =
 view : Model -> Html Msg
 view model =
     nav []
-        [ img [ src "assets/backdrop.jpg", css [ width (pct 100) ] ] []
+        [ img [ src "assets/backdrop.jpg", css [ width (pct 100) ], class "some-whatever-class" ] []
         , btn [ onClick DoSomething ] [ text "Click me!" ]
-        , StyledSvg.svg [ SvgAttribs.css [], SvgAttribs.width "100", SvgAttribs.height "100" ] [ StyledSvg.circle [ SvgAttribs.cx "50", SvgAttribs.cy "50", SvgAttribs.r "40" ] [] ]
+        , StyledSvg.svg [ SvgAttribs.class "some-whatever-NS-class", SvgAttribs.css [ opacity (num 0.5) ], SvgAttribs.width "100", SvgAttribs.height "100" ] [ StyledSvg.circle [ SvgAttribs.cx "50", SvgAttribs.cy "50", SvgAttribs.r "40" ] [] ]
```

Comment:
The circle is a bit low-contrast against the background.

How would you feel about using bright red instead?

```suggestion
         , StyledSvg.svg
            [ class "some-whatever-NS-class"
            , SvgAttribs.css [ color (rgb 256 0 0) ]
            , SvgAttribs.width "100"
            , SvgAttribs.height "100"
            ]
            [ StyledSvg.circle
                [ SvgAttribs.cx "50"
                , SvgAttribs.cy "50"
                , SvgAttribs.r "40"
                , SvgAttribs.fill "currentcolor"
                ]
                []
            ]
```

Reply:
This is fantastic, many thanks for opening the PR I shall review and merge post haste. 

On your points -

- The changes I made definitely only ever deal with NodeNS and KeyedNodeNS types so I think we are good there.
We are limited in testing this in the package with elm-test but I don't see why we couldn't test with something like  [playwright](https://playwright.dev/). I did this for elm-select and it works great. Perhaps a little overkill but we could raise an issue to explore how to test the svg class behaviour.

- The current runtime exception regarding the svg styles already exists so whilst we haven't fixed the issue, we haven't made it worse.

- I think it makes sense to patch elm-test to handle svg classes.

Reply:
Yay, glad you like it!

- I haven't ever used playwright -- looks great! Maybe [Percy](https://percy.io/) could be neat to have in place too? 

- True! 🙈 

- No, as far as I can tell, I don't think elm-test with SVG classes was ever working 🤔

Reply:
Looks like an issue in elm-test exists https://github.com/elm-explorations/test/issues/134. 

Had a super quick look at the project and I think a fix is relatively straightforwards. (I will regret saying that)